fuel/schemes.py

```diff
@@ -1,10 +1,9 @@
 from abc import ABCMeta, abstractmethod
 
 import numpy
-from picklable_itertools import chain, repeat, imap, _iter
+from picklable_itertools import chain, repeat, imap, xrange, iter_
```

Comment:
Mm... I'm undecided. The thing is, the main selling point seems to be the ability to use the `for ... in xrange(...)` construct. The thing is, you would never pickle in the middle of a `for`-loop, since the iterator is implicit in that case it won't be serialized, and you would lose your position within the loop. Also, if someone ends up using this for a very large number of examples for very short iterations, the additional overhead might actually become noticeable (or maybe not?). On the other hand, maybe it's better to stay on the safe side and use it? I really don't know, so fine either way.